examples/flutter_gallery/lib/gallery/app.dart

```diff
@@ -93,23 +93,27 @@ class GalleryAppState extends State<GalleryApp> {
         });
       },
       showPerformanceOverlay: _showPerformanceOverlay,
-      onShowPerformanceOverlayChanged: widget.enablePerformanceOverlay ? (bool value) {
+      onShowPerformanceOverlayChanged: widget.enablePerformanceOverlay ? (
+          bool value) {
```

Comment:
nit: please don't wrap like this :-)

Reply:
(is that what dartfmt does?)

Reply:
No, I was editing in Intellij, didn't do any intentional reformatting, I thought maybe this was a Flutter plugin specific thing.  I'll back these out if we want to try to move forward with this.